tester_app/app/views/hello_world/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Client Rendering</h1>
+<%= react_component("HelloWorldApp", {initialName: "Stranger"}, prerender: false, trace: true) %>
```

Comment:
@justin808 You're saying you want me to make a `hello_world_controller` `#index` action that sets `@hello_world_props = { initialName: "Stranger"}`? 

As far as linting, it may have been a false negative with linting because it has a `tt` extension, but fixed.